server/main.lua

```diff
@@ -66,8 +66,8 @@ AddEventHandler('qb-garbagejob:server:PayShit', function(amount, location)
             end
         end
 
-        TriggerClientEvent('QBCore:Notify', src, "You have $"..amount..",- got paid to your bank account!", "success")
+        TriggerClientEvent('QBCore:Notify', src, "You have earned $"..amount.." dollars, it was paid into your bank account!", "primary")
```

Comment:
TriggerClientEvent('QBCore:Notify', src, "You have earned $"..amount..". It has been paid into your bank account!", "primary")